roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -76,11 +76,19 @@ def __init__(self, *args, **kwargs):
         if oreg_auth_user != '' and oreg_auth_password != '':
             oreg_auth_user = self.template_var(oreg_auth_user)
             oreg_auth_password = self.template_var(oreg_auth_password)
-            self.skopeo_command_creds = "--creds={}:{}".format(quote(oreg_auth_user), quote(oreg_auth_password))
+            self.skopeo_command_creds = quote("--creds={}:{}".format(oreg_auth_user, oreg_auth_password))
 
         # record whether we could reach a registry or not (and remember results)
         self.reachable_registries = {}
 
+        # take note of any proxy settings needed
+        proxies = []
```

Comment:
When building variables like this, it's nice to add some comments like
#check openshift_http_proxy openshift_https_proxy openshift_no_proxy
so users/devs can grep for them.